page01_index.php

```diff
@@ -52,118 +53,114 @@
-						if (slideIndex > x.length) {slideIndex = 1}
-						x[slideIndex-1].style.display = "block";
-						setTimeout(carousel, 2000); // Change image every 2 seconds
-					}
-				</script>
+						<!-- Playlist photos / Queries -->
+						<!-- Responsive Grid. Four columns on tablets, laptops and desktops. Will stack on mobile devices/small screens (100% width) -->
 
-				<br>
-				<br>
+						<div class="w3-row-padding w3-center">
+							<div class="w3-col m3">
+								<a href="Playlist_Genre.rock .php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="ROCK"> </a>
+							</div>
 
-				<!-- Playlist photos / Queries -->
-				<!-- Responsive Grid. Four columns on tablets, laptops and desktops. Will stack on mobile devices/small screens (100% width) -->
 
-				<div class="w3-row-padding w3-center">
-					<div class="w3-col m3">
-						<a href="Playlist_Genre.rock .php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="ROCK"> </a>
-					</div>
+							<div class="w3-col m3">
+								<a href="Playlist_Genre.pop.php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="POP"> </a>
+							</div>
 
 
-					<div class="w3-col m3">
-						<a href="Playlist_Genre.pop.php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="POP"> </a>
-					</div>
 
+							<div class="w3-col m3">
+								<a href="Playlist_Genre.country.php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="COUNTRY"> </a>
+							</div>
 
+							<div class="w3-col m3">
+								<a href="Playlist_Genre.jazz.php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="COUNTRY"> </a>
+							</div>
+						</div>
 
-					<div class="w3-col m3">
-						<a href="Playlist_Genre.country.php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="COUNTRY"> </a>
-					</div>
+		
 
-					<div class="w3-col m3">
-						<a href="Playlist_Genre.jazz.php"> <img src="images/placeholder.jpg" class="w3-button w3-round w3-opacity w3-hover-opacity-off" style="width:100%"  alt="COUNTRY"> </a>
-					</div>
+					</div>	
 				</div>
-
-				<br>
-				<br>
-				<br>
-
-			</div>	
+			</div>
 		</div>
-
+		<br>
+		<br>
+		<br>
+		<br>
 
 		<!-- Footer -->
-		<footer class="w3-bottomfooter w3-center w3-black w3-padding-16">
-			<p1>Created by <a target="_blank" class="w3-hover-text-orange">Soundwave</a> &copy; Copyright Liv Amer 2022</p1>
-			<br>
-			<br>
-			<a href="#top" class="back-to-top-link w3-button w3-light-grey"><i class="fa fa-arrow-up w3-margin-right"></i>To the top</a>
+		<footer class="w3-bottom w3-center w3-grey w3-padding-16">
```

Comment:
change font writing to personalise the footer, black arial writing with grey theme and padding of 16